src/faebryk/libs/picker/api/pickers.py

```diff
@@ -43,15 +47,23 @@ def add_api_pickers(module: Module, base_prio: int = 0) -> None:
     # TODO: might want to remove this 'if' since the trait might be defined later
     # but for now handy to guard, since it makes logging more concise
     if module.has_trait(F.has_descriptive_properties):
```

Comment:
now only adds picker if if has the props, also doesnt add db picker for explicit stuff